Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -21,6 +21,8 @@
 SERVER = demisto.params().get('url', '').rstrip('/')
 USERNAME = demisto.params().get('credentials', {}).get('identifier')
 PASSWORD = demisto.params().get('credentials', {}).get('password')
+API_KEY = demisto.params().get('api_key')
```

Comment:
we should add a test of allowing only one of them to be entered, API or Username password.
Is it even possible with a credentials box? i think that it might be required

Reply:
The main issue with a credentials box is that we can only name the first display name, but that's not an issue because I can use an encrypted box. I could use it for both the API Key and the ID, what do you think?